src/main/java/techcourse/myblog/service/CommentService.java

```diff
@@ -0,0 +1,48 @@
+package techcourse.myblog.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import techcourse.myblog.domain.article.Article;
+import techcourse.myblog.domain.comment.Comment;
+import techcourse.myblog.domain.comment.CommentException;
+import techcourse.myblog.domain.user.User;
+import techcourse.myblog.dto.CommentDto;
+import techcourse.myblog.repository.CommentRepository;
+
+@Service
+@Transactional
+public class CommentService {
+
+    private final CommentRepository commentRepository;
+    private final UserService userService;
+    private final ArticleService articleService;
+
+    public CommentService(CommentRepository commentRepository, UserService userService, ArticleService articleService) {
+        this.commentRepository = commentRepository;
+        this.userService = userService;
+        this.articleService = articleService;
+    }
+
+    public Comment addComment(Long articleId, String email, CommentDto commentDto) {
```

Comment:
참조 자료형 대신 기본 자료형을 사용하면 어떨까요? 아래의 글을 참고하세요.
https://jaehun2841.github.io/2019/03/01/effective-jave-item61

```suggestion
    public Comment addComment(long articleId, String email, CommentDto commentDto) {
```